core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4015,6 +4035,14 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String USER_WORKER_LIST_REFRESH_INTERVAL =
         "alluxio.user.worker.list.refresh.interval";
 
+    //
+    // Shell command related properties
+    //
+    public static final String SHELL_COPY_FROM_LOCAL_BUFFER_SIZE =
```

Comment:
this property seems related to "alluxio.user.file.copyfromlocal.write.location.policy.class" but is under a very different namespace. i prefer "alluxio.user.file.copyfromlocal.buffer.size" to be more consistent or an option to the CLI like "-bs"

Reply:
`-bs` sounds good, PTAL.